hedera-node/src/test/java/com/hedera/services/txns/token/TokenDissociateTransitionLogicTest.java

```diff
@@ -1,207 +1,146 @@
 package com.hedera.services.txns.token;
 
-/*-
```

Comment:
nit: I think that the license was removed by mistake

Reply:
Restored.